src/layout/fontSubstitution/index.js

```diff
@@ -1,32 +1,78 @@
 import { pathOr, last } from 'ramda';
 
+import Font from '../../font';
 import StandardFont from './standardFont';
 
 const fontCache = {};
 
 const IGNORED_CODE_POINTS = [173];
 
-const getFontSize = pathOr(12, ['attributes', 'fontSize']);
-
-const getFallbackFont = () => {
-  return getOrCreateFont('Helvetica');
+const STANDARD_FONT_ALIASES = {
+  'sans-serif': 'Helvetica',
+  serif: 'Times-Roman',
+  monospace: 'Courier',
 };
 
-const getOrCreateFont = name => {
-  if (fontCache[name]) return fontCache[name];
-
-  const font = new StandardFont(name);
-  fontCache[name] = font;
+const getFontSize = pathOr(12, ['attributes', 'fontSize']);
 
-  return font;
+const getFallbackFont = () => {
+  return getFontStack({ font: 'Helvetica' })[0];
 };
 
-const shouldFallbackToFont = (codePoint, font) => {
-  return (
-    !IGNORED_CODE_POINTS.includes(codePoint) &&
-    !font.hasGlyphForCodePoint(codePoint) &&
-    getFallbackFont().hasGlyphForCodePoint(codePoint)
-  );
+const toFontNameStack = (...fontFamilyObjects) =>
+  fontFamilyObjects
+    .map(fontFamilies =>
+      typeof fontFamilies === 'string'
+        ? fontFamilies.split(',').map(f => f.trim())
+        : Array.from(fontFamilies || []),
+    )
+    .flat()
```

Comment:
This broke for me on nodejs. Is it possible to use `flatten` from `ramda` instead?